Trimble.Modus.Components/Controls/TMInput/BaseInput.xaml.cs

```diff
@@ -87,10 +97,21 @@ public partial class BaseInput : ContentView
     /// Gets or sets value that indicates whether the input control is readonly or not.
     /// </summary>
     public static readonly BindableProperty IsReadOnlyProperty =
-        BindableProperty.Create(nameof(IsReadOnly), typeof(bool), typeof(BaseInput), false, propertyChanged: OnReadOnlyPropertyChanged);
+        BindableProperty.Create(nameof(IsReadOnly), typeof(bool), typeof(BaseInput), false, propertyChanged: OnReadOnlyPropertyChanged);
+
+    /// <summary>
+    /// CommandProperty for Focused Event
+    /// </summary>
+    public static readonly BindableProperty FocusedEventCommandProperty =

```

Comment:
Should be just 'FocusedCommentProperty' and `UnFocusedCommandProperty`

Reply:
Renamed